css/style.css

```diff
@@ -0,0 +1,682 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: Gotham Pro, sans-serif;
+}
+
+body {
+  min-width: 1170px;
+  margin-top: 10px;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1030px;
+}
+
+/* Первый блок */
+
+.heading {
+  min-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 9px;
+}
+
+.heading__nav-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.heading__logo {
+  margin-top: 4px;
+  margin-right: 45px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000;
+  margin-right: 30px;
+}
+
+.navigation__link:hover {
+  color: #b5b5b5;  
+}
+
+.navigation__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0,0,0,0.5);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)